src/EDA/plot_EDA.py

```diff
@@ -0,0 +1,51 @@
+import matplotlib.pyplot as plt
+
+def plot_class_per_image_hist(df_train):
+    """
+    Function that plots
+
+
+    Parameters
+    ----------
+    df_train: DataFrame
+         Data Frame with the run length encoded segmentations for each image-label
+         pair in the train_images (provided by Kaggle)
+
+    Returns
+    -------
+    Plots:
+        Two histograms: 1) histogram of label patterns per image_id
+                        2) histogram of number of labels per image_id
+
+    """
+    from get_train_info import get_class_per_image
```

Comment:
Comment on style: There is almost never a good reason to have an import statement buried in a function. Please put it at the top of the script